packages/opentelemetry-api/src/metrics/NoopMeter.ts

```diff
@@ -57,8 +60,25 @@ export class NoopMeter implements Meter {
    * Returns constant noop observer.
    * @param name the name of the metric.
    * @param [options] the metric options.
+   * @param [callback] the observer callback
    */
-  createObserver(name: string, options?: MetricOptions): Observer {
+  createObserver(
```

Comment:
this should be renamed to `createValueObserver` right?

Reply:
but probably not is this PR - there is a separate issue for that -> https://github.com/open-telemetry/opentelemetry-js/issues/1113

Reply:
unless it make sense to make it in the same PR, as I have to wait anyway until this PR will be merged, WDYT ?

Reply:
I'm ok either way

Reply:
just pushed the renaming